cookbook/configuration/web_server_configuration.rst

```diff
@@ -48,6 +48,12 @@ are:
             Allow from All
         </Directory>
 
+        # uncomment the following lines if you install assets as symlinks
+        # or run into problems when compiling LESS/Sass/CoffeScript assets
+        # <Directory /var/www/project>
```

Comment:
shouldn't you suggest this change in here too https://github.com/symfony/symfony-standard/blob/2.7/web/.htaccess ?